packages/atomic/src/locales.json

```
@@ -700,32 +700,32 @@
     "fr": "Segments d'URI réduits"
   },
   "disconnected": {
-    "en": "Could not connect.",
-    "fr": "Impossible de se connecter."
+    "en": "No Internet access.",
+    "fr": "Pas d'accès internet."
   },
   "check-your-connection": {
     "en": "Your query couldn't be sent. Verify your internet connection.",
     "fr": "Votre requête n'a pas plus être envoyée. Peut-être que vous avez perdu votre connexion internet."
   },
   "no-endpoints": {
-    "en": "The Coveo Organization has no registered endpoints.",
-    "fr": "L'organisation Coveo n'a pas de point de terminaison unique."
+    "en": "Your organization {{org}} has no available content.",
+    "fr": "Votre organisation {{org}} n'a pas contenu disponible."
   },
   "invalid-token": {
-    "en": "The token is invalid.",
-    "fr": "Le jeton d'identification est invalide."
+    "en": "Ensure that the token is valid.",
+    "fr": "Faites en sorte que le jeton d'identification soit invalide."
   },
   "add-sources": {
-    "en": "You will need to add sources in your index, or wait for the created sources to finish indexing.",
+    "en": "Add content sources or wait for your newly created sources to finish indexing.",
     "fr": "Vous devrez ajouter des sources à votre index, ou attendre la fin de l'indexation des sources créées."
```

This translation doesn't seem perfectly equivalent. What about:
```suggestion
    "fr": "Ajoutez des sources de contenu ou attendez la fin de l'indexation de vos nouvelles sources."
```